### ISSUE TITLE
Fallback SVG image should not get rasterized even if WebP detection is enabled and User-Agent has indicated WebP support

### ISSUE THREAD

**Issue by pfyod** [imgproxy/imgproxy#609]
**Expected**: if fallback image is SVG, it should be passed through as-is. 
**Actual**: if WebP detection is enabled and user agent supports WebP, SVG fallback images get rasterized (+resized, etc)

**Test:**
```bash
docker run --rm \
  -e IMGPROXY_FALLBACK_IMAGE_DATA=`echo '<svg xmlns="http://www.w3.org/2000/svg"><text>SVG</text></svg>' | base64 -w 0` \
  -e IMGPROXY_ENABLE_WEBP_DETECTION=true \
  -p 8080:8080 darthsim/imgproxy:v2.16.3

curl -sI -X GET -H "Accept: image/webp,*/*" \
  http://localhost:8080/plain/rs:fill:1000:1000/plain/http://example.com/404.image \
  | grep Content-Type
```
→ `Content-Type: image/webp`

vs

```bash
docker run --rm \
  -e IMGPROXY_FALLBACK_IMAGE_DATA=`echo '<svg xmlns="http://www.w3.org/2000/svg"><text>SVG</text></svg>' | base64 -w 0` \
  -e IMGPROXY_ENABLE_WEBP_DETECTION=false \
  -p 8080:8080 darthsim/imgproxy:v2.16.3

curl -sI -X GET -H "Accept: image/webp,*/*" \
  http://localhost:8080/plain/rs:fill:1000:1000/plain/http://example.com/404.image \
  | grep Content-Type
```
→ `Content-Type: image/svg+xml`

**Comment by DarthSim** [comment]
Hey!

It's not only fallback image behavior. If you try to process SVG with WebP detection enabled, you'll get it converted to WebP (if the browser supports it). If you use the OSS version of imgproxy (thus you don't have SVG minimization feature), you can just disable SVG processing with [IMGPROXY_SKIP_PROCESSING_FORMATS](https://docs.imgproxy.net/#/configuration?id=skip-processing) config.

However, I think this behavior should probably be changed.

**Comment by pfyod** [comment]
@DarthSim: thanks for the tip! Will just disable processing for `svg`.

> If you try to process SVG with WebP detection enabled, you'll get it converted to WebP (if the browser supports it)

The difference to the fallback image case is that one can control the conversion behavior by adding the `.svg` extension to the generated URL; we already do that to have both, WebP conversion for raster images and passthrough for SVG. Obviously, can only do that if the source is known to be SVG at the time of URL generation.

**Comment by DarthSim** [comment]
Hey @pfyod!
This issue has been fixed in v3.0.0.beta1.

You can try the OSS version of `v3.0.0.beta1`, or you can subscribe to our imgproxy Pro v3 [Early Access program](https://www.producthunt.com/upcoming/imgproxy)
